apt: fix KeyError for deb=file

resmo opened the pull request:
##### ISSUE TYPE
- Bugfix Pull Request
##### COMPONENT NAME

packaging/os/apt.py
##### ANSIBLE VERSION

```
2.1 >=
```
##### SUMMARY

Fix KeyError: 'prepared' while installing dependencies using deb=<file>.deb
This error shows up when --diff was not passed by and the deb files has dependencies not yet installed.
Closes #3752.

gregdek commented:
Thanks @resmo for this PR. This module is maintained by the Ansible core team, so it can take a while for patches to be reviewed. Thanks for your patience.

Core team: please review according to guidelines (http://docs.ansible.com/ansible/developing_modules.html#module-checklist) and comment with 'needs_revision' or merge as appropriate.

[This message brought to you by your friendly Ansibull-bot.]

resmo committed:
apt: fix KeyError for deb=file

Fix KeyError: 'prepared' while installing dependencies using deb=<file>.deb
This error shows up when --diff was not passed by and the deb files has dependencies not yet installed.
Closes #3752.

resmo commented:
shipit
